bot/seasons/easter/egg_facts.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+
+class EasterFacts(commands.Cog):
+    """A cog for easter egg facts."""
+    def __init__(self, bot):
+        self.bot = bot
+        self.facts = self.load_json()
+
+    @staticmethod
+    def load_json():
+        """Loading the json data"""
+        p = Path('bot', 'resources', 'easter', 'easter_egg_facts.json')
+        with p.open(encoding="utf8") as f:
+            facts = load(f)
+        return facts
+
+    async def background(self):
```

Comment:
Do you think it would be nice to have a more descriptive method name? Such as

```suggestion
    async def send_egg_fact_daily(self):
```